lib/chef/resource/windows_task.rb

```diff
@@ -464,6 +461,609 @@ def sec_to_dur(seconds)
       def sec_to_min(seconds)
         seconds.to_i / 60
       end
+
+      action_class do
+        if ChefUtils.windows_ruby?
+          include ::Win32
```

Comment:
?

Reply:
that's from the original code

Reply:
yeah I see the comment above. That's some serious wtf there